### PR TITLE
Handle portalIP 'None' and empty service.spec.ports

### PR DESCRIPTION
**Pull request by spadgett**
Update the Project and Browse -> Services pages to handle portalIP value
'None' and an empty ports array in service.spec.

See also 8a2ba7058aeaa89b92f8d668a76f225c51257bc4

### CODE REVIEW

**Comment by spadgett** [comment]
@jwforres @liggitt please review

File: assets/app/views/services.html [hunk 1]
```diff
@@ -11,8 +11,12 @@
...
         <h3>{{service.metadata.name}}</h3>
         <div>Created: <relative-timestamp timestamp="service.metadata.creationTimestamp"></relative-timestamp></div>
-        <div>IP: {{service.spec.portalIP}}</div>
+        <div>IP:
+          <span ng-if="service.spec.portalIP == 'None'"><em>none</em></span>
+          <span ng-if="service.spec.portalIP != 'None'">{{service.spec.portalIP}}</span>
+        </div>
         <div>Ports:</div>
+        <div ng-if="!service.spec.ports.length"><em>none</em></div>
         <ul>
```
Comment (liggitt) [inline assets/app/views/services.html]: does an empty <ul></ul> do anything weird padding-wise if there are no ports?

Comment (spadgett) [inline assets/app/views/services.html]: It looked OK, but probably better not to have it anyway.... I'll fix.

**Comment by jwforres** [comment]
other than @liggitt's one comment this LGTM, go ahead and add bindata

**Comment by liggitt** [comment]
[merge]

**Comment by liggitt** [comment]
spurious, [merge]

**Pull request merged**